src/tests/youtube/Youtube_EME_001.test.js

```diff
@@ -0,0 +1,119 @@
+import {
+  pluginDeactivate,
+  webKitBrowserStartAndResume,
+  setWebKitUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
+
+let url =
+  'http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2016.html?test_type=encryptedmedia-test&enablewebm=off&command=run&tests=1,6,17,18'
+
```

Comment:
Also we need to only run the playready tests AFAIK

Reply:
Updated the URL to 2018